application/helpers/update/updates/Update_486.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+use LsDefaultDataSets;
+
+class Update_486 extends DatabaseUpdateBase
+{
+    public function up()
+    {
+        $this->db->createCommand()->update(
+            "{{question_themes}}",
+            [
+                'xml_path' => 'themes/question/bootstrap_buttons_multi/survey/questions/answer/multiplechoice',
+                'image_path' => '/themes/question/bootstrap_buttons_multi/survey/questions/answer/multiplechoice/assets/bootstrap_buttons_multiplechoice.png',
```

Comment:
That's correct, xml_path starts with letter, image_path starts with slash?

Reply:
shouldn't we just straitght that (in another story)?
Maybe add a function which will allow any kind of path and then converted to the required type of path?